atlas/packages/app/src/shared/components/Checkbox/Checkbox.tsx

```diff
@@ -25,7 +25,8 @@ export default function Checkbox({
       <div css={styles.outerContainer}>
         <div css={styles.innerContainer}>
           <input css={styles.input} type="checkbox" checked={selected} disabled={disabled} onChange={onChange} />
-          {selected && (icon === 'check' ? <CheckIcon css={styles.icon} /> : <DashIcon css={styles.icon} />)}
+          {selected &&
+            (icon === 'check' ? <Icon icon="check" css={styles.icon} /> : <Icon icon="dash" css={styles.icon} />)}
```

Comment:
Since it's the same component now it'd be more readable to write it as:
```tsx
{selected && <Icon icon={icon === 'check' ? 'check' : 'dash'} css={styles.icon} />}
```